.github/workflows/integration-test.yml

```diff
@@ -15,12 +15,12 @@ jobs:
 
       - name: Download OpenSearch Core
         run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.7/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-1.3.7-linux-x64.tar.gz
+          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.8/7003/linux/x64/tar/builds/opensearch/dist/opensearch-min-1.3.8-linux-x64.tar.gz
```

Comment:
This will temporarily change from latest -> 7003. As of the time of writing, `latest` resolves to 7000 and a SNAPSHOT of the security plugin is not available for that build.